doc/common_installation_problems/README.md

```diff
@@ -265,29 +265,6 @@ PassThroughPattern: (packages\.gitlab\.com|packages-gitlab-com\.s3\.amazonaws\.c
 
 Read more about `apt-cacher-ng` and the reasons why this change is needed [on the packagecloud blog](http://blog.packagecloud.io/eng/2015/05/05/using-apt-cacher-ng-with-ssl-tls/).
 
-### Using self signed certificate or custom certificate authorities
```

Comment:
@bbodenmiller If we remove the header we break any existing links all around the internet. Can you leave the header and state where this section was moved?